blocks/feeds-source-content-api-by-day2-block/README.md

```diff
@@ -0,0 +1,39 @@
+# Content-API By Day2 content source
+
+Creates an ElasticSearch DSL syntax to query Content-API limited to a single day. This is
+for sitemap/YYYY-MM-DD formats with 1 hour TTL (3600).
+
+## Globals
+
+- CONTENT_BASE
+- feedDefaultQuery
+
+## Parameters
+
+- dateField: ANS date field to use in range statement
+- dateRange: date to use in YYYY-MM-DD format or 'latest' for today. latest is use instead of generating todays date because of the chance that the process making the request (fusion) is in a different tz then the ES cluster.
```

Comment:
dateRange: date to use in YYYY-MM-DD format or 'latest' for today. latest is used instead of generating today's date because of the chance that the process making the request (fusion) is in a different TZ than the ES cluster.

Reply:
Updated all three README.md's